metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/AthenaAggregator.java

```diff
@@ -259,4 +258,23 @@ protected SelectConditionStep<Record> createUnnestQueryWithModifiedTime(AthenaTa
                 .from(unnestedExecutionsWithFileModifiedTime)
                 .where(FILE_MODIFIED_TIME_ROW_NUM_FIELD.eq(inline(1)));
     }
+
+    private Condition createPartitionSelector(AthenaTablePartition partition) {
+        return createFieldSelector(ENTITY_FIELD, partition.entity())
+            .and(createFieldSelector(REGISTRY_FIELD, partition.registry()))
+            .and(createFieldSelector(ORG_FIELD, partition.org()))
+            .and(createFieldSelector(NAME_FIELD, partition.name()))
+            .and(createFieldSelector(VERSION_FIELD, partition.version()));
+    }
+
+    private Condition createFieldSelector(Field<String> field, Set<String> values) {
+        // Use the "equals" syntax when there's one value to compare to, just in case
```

Comment:
Very thorough, but would have thought there would not be a difference. 
(can leave since you already have it)